README.md

```diff
@@ -1,5 +1,7 @@
 # SCIFIO ImageIO plugin
 
+![](https://github.com/KitwareMedical/MultipleImageIterator/workflows/Build,%20test,%20package/badge.svg)
```

Comment:
Badge URL needs to be updated for this repository.

Reply:
The badge URL has been fixed, thank you for catching this!